pkg/data/eventformatters.go

```diff
@@ -1,4 +1,4 @@
-package lib
+package data
 
 import "fmt"
 
```

Comment:
Really this logic should be a part of the ceilometer-event handler. For this to work, I'll have to rework the event interface

Reply:
Yeah I agree that given the fact that only ceilometer events need formatting, it might seem like we need formatters only in ceilometer-events handler, but we gonna need formatting for collectd-events when sensubility functionality will be added.

Reply:
Also sg-agent data source will be another data source event, so we have to look to the future.

Reply:
Yes if we decide to keep sending sensubility data to the `collectd/telemetry` address, the collectd-events handler will have to handle them.
sg-agent messages should be handled by a separate handler